tests/Unit/DataStructures/Test_Matrix.py

```diff
@@ -40,5 +40,13 @@ def assignment_test():
         self.assertRaises(RuntimeError, assignment_test)
 
 
+    def test_to_numpy_array(self):
+        M = Matrix(2,2)
+        M[0,0] = 1.0
+        M[0,1] = 2.0
+        A = M.to_numpy_array()
+        self.assertEquals(A[0,0], 1.0)
```

Comment:
I think you can summarize all three asserts as `type(A) is numpy.ndarray`, or something similar